pkg/client/packet/client.go

```diff
@@ -0,0 +1,33 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Delete this file (the whole `pkg/client/packet` directory).

Reply:
Done